templates/rome_templates/index.html

```diff
@@ -14,6 +14,7 @@
 			<li><a href="essays">Essays</a></li>
 			<li><a href="links">Links</a></li>
 			<li><a href="about">About</a></li>
+			<!--<li><a href="search">Search</a></li>-->
```

Comment:
This commented line puts a search tab on the index page which leads to the search page that is currently in progress. I just commented it out because the search page is not yet complete so I didn't want the tab to show up on the dev server. I'll uncomment it  when the search page is done.